python/lsst/summit/extras/logUtils.py

```diff
@@ -44,12 +44,13 @@ class LogBrowser():
     -----
     Many tasks throw errors with values in them, meaning the ``doFailZoology``
     function doesn't collapse them down to a single failure case as one would
-    like. If this is the case, the first part of the message that is common
-    amongst the ones you would like to be classed together, and add it to the
-    class property ``SPECIAL_ZOO_CASES`` to declare a new type of error animal.
+    like. If this is the case, take the first part of the message that is
+    common among the ones you would like to be classed together, and add it to
+    the class property ``SPECIAL_ZOO_CASES`` to declare a new type of error
+    animal.
 
     example usage:
```

Comment:
You can put this into its own Numpydoc section, i.e.:
```
Examples
--------
```

Reply:
The reason I didn't want to do that is that sometimes those are actually considered literally runnable, and this one certainly isn't and could not really ever be, realistically, given the need for a butler and a collection.